pkg/windows/windows.go

```diff
@@ -170,6 +170,8 @@ type windows struct {
 	// workerIgnitionEndpoint is the Machine Config Server(MCS) endpoint from which we can download the
 	// the OpenShift worker ignition file.
 	workerIgnitionEndpoint string
+	// clusterDNSIP is the IP address used to configure the kubelet with a specific clusterDNS IP
```

Comment:
Borrowing @sebsoto's suggestion on the WMCB side https://github.com/openshift/windows-machine-config-bootstrapper/pull/306#discussion_r740263434